src/SwedbankPay.Sdk.Infrastructure/PaymentInstruments/PricesDto.cs

```diff
@@ -0,0 +1,24 @@
+﻿using SwedbankPay.Sdk.PaymentInstruments.Card;
+using System;
+using System.Collections.Generic;
+
+namespace SwedbankPay.Sdk.PaymentInstruments
+{
+    public class PricesDto : Identifiable

```

Comment:
`Identifiable` shouldn't be used by DTO's, I think. DTO's should be as stupid as possible and with as little inheritance and such as possible. If we want subclassing in the DTO object hierarchy, we should only inherit from other DTO objects and thus need an `IdentifiableDto`. But I don't think that's very useful.